membership_initial_fee/models/account_move_line.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2015-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Onestein - Andrea Stirpe
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0
+
+from odoo import api, models
+from odoo.tools.translate import _
```

Comment:
Could you import "_" in the previous import line?

Reply:
Done